indico/modules/rb_new/client/js/modules/bookRoom/BookRoomModal.jsx

```diff
@@ -553,6 +554,10 @@ class BookRoomModal extends React.Component {
                                            placeholder={Translate.string('Reason for booking')}
                                            disabled={bookingBlocked(fprops)}
                                            required />
+                                    <Overridable id="BookRoomModal.extraFields"
+                                                 disabled={bookingBlocked(fprops)}
+                                                 form={fprops.form}
```

Comment:
This is a workaround until we start sharing the same `react` bundle between Indico and the plugins.